### PR TITLE
feat: Change ids to integer to use less space

### PR DESCRIPTION
**Pull request by john-gom**
### What
Changing the product ids from UUIDs to integer (allocated from a sequence)

### Part of 
- #39

### CODE REVIEW

File: .github/workflows/container-deploy.yml [hunk 1]
```diff
@@ -28,26 +28,39 @@ jobs:
...
           echo "COMMON_NET_NAME=common_net" >> $GITHUB_ENV
+
           # mongodb and redis (through stunnel)
           echo "MONGO_URI=mongodb://10.1.0.102:27017" >> $GITHUB_ENV
           echo "REDIS_URL=redis://10.1.0.122:6379" >> $GITHUB_ENV
+
+          # Postgres config
           echo "POSTGRES_SHM_SIZE=1g" >> $GITHUB_ENV
+          echo "QUERY_POSTGRES_SHARED_BUFFERS=16GB" >> $GITHUB_ENV
+          echo "QUERY_POSTGRES_WORK_MEM=1GB" >> $GITHUB_ENV
```
Comment (alexgarel) [inline .github/workflows/container-deploy.yml:63]:
```suggestion
          # we use memory to gain performance on reads and writes
          echo "QUERY_POSTGRES_SHARED_BUFFERS=16GB" >> $GITHUB_ENV
          echo "QUERY_POSTGRES_WORK_MEM=1GB" >> $GITHUB_ENV
```

File: src/migrations/Migration20240614163139.ts
Comment (alexgarel) [inline src/migrations/Migration20240614163139.ts:1]: I don't understand this long migration file.

**Review by alexgarel — COMMENTED** [review]
LGTM for the migrations, I don't understand but I imagine you know how it works :-)

**Comment by alexgarel** [comment]
Thanks @john-gom

File: src/migrations/Migration20240614163139.ts
Comment (john-gom) [reply src/migrations/Migration20240614163139.ts:1]: Added some comments

**Pull request merged**